### ISSUE TITLE
Environment.ts gets merged faulty (using CobiGen generate)

### ISSUE THREAD

**Issue by jan-vcapgemini** [devonfw/devon4ng-application-template#189]
### Expected behavior

As a CobiGen user, I want to generate an angular client project from my Eto file.

Result of merged src/environments/environment.ts should look like:

```
export const environment = {
  production: false,
  restPathRoot: 'http://localhost:8081/',
  restServiceRoot: 'http://localhost:8081/services/rest/',
  security: 'jwt',
};
```

### Actual behavior

After generating the project by following the steps on:
https://github.com/devonfw/cobigen/wiki/howto_angular-client-generation
The merged src/environments/environment.ts file gets merged with an invalid format.

Result looks like this:
```
export const environment: = {
    production: false,
    restPathRoot: 'http://localhost:8081/',
    restServiceRoot: 'http://localhost:8081/services/rest/',
    security: 'jwt'
};
```

### Steps to reproduce (bug) / Use Case of feature request (enhancement)

1. Follow steps on: https://github.com/devonfw/cobigen/wiki/howto_angular-client-generation using the [my-thai-star](https://github.com/devonfw/my-thai-star) project
2. Generate from my-thai-star\java\mtsj\api\src\main\java\com\devonfw\application\mtsj\general\logic\api\to\BinaryObjectEto.java

### Affected version:

* CobiGen Eclipse Plug-in	7.1.0.202012101009
* CobiGen Eclipse Plug-in	7.0.0
* CobiGen - TypeScript Plug-in 7.1.0

**Comment by maybeec** [comment]
As agreed with @ssarmokadam and @sujith-mn, we can revert that back when we fixed devonfw/cobigen#1291.
Until now, we remove it to make E2E experience smoothly working.

**Comment by maybeec** [comment]
@jan-vcapgemini could you provide a PR here as well?
